### ISSUE TITLE
Support bpmn-js v7

### ISSUE THREAD

**Issue by asoahc** [bpmn-io/bpmn-js-cli#7]
As defined in the [`package.json`](https://github.com/bpmn-io/bpmn-js-cli/blob/master/package.json) the the library does not support `bpmn-js@7` yet.

When I install it, a warning is logged:

```
npm WARN bpmn-js-cli@1.2.0 requires a peer of bpmn-js@^1.x || ^2.x || ^3.x || ^4.x || ^5.x || ^6.x but none is installed. You must install peer dependencies yourself.
```

Can you upgrade dependencies? Thanks a lot.

**Comment by philippfromme** [comment]
Closed via https://github.com/bpmn-io/bpmn-js-cli/commit/001dc3477665c4bf8520f152383e49c3b2259770

**Comment by asoahc** [comment]
Many Thanks

**Comment by asoahc** [comment]
Why was the version in package.json not increased?

**Comment by philippfromme** [comment]
Because we haven't released a new version, yet.

**Comment by philippfromme** [comment]
We released v1.3.0: https://www.npmjs.com/package/bpmn-js-cli/v/1.3.0

**Comment by philippfromme** [comment]
I realized that the CLI is actually using `saveSvg` and `saveXML`. We will release a patch version that uses promises instead of callbacks. Sorry for the inconvenience.
